functions/.testenv

```diff
@@ -1,2 +1,4 @@
 GCLOUD_PROJECT="demo-rally"
-FIRESTORE_EMULATOR_HOST="localhost:8080"
\ No newline at end of file
+FIRESTORE_EMULATOR_HOST="localhost:8080"
+FIREBASE_EMULATOR_HUB="localhost:4400"
+NODE_OPTIONS=--experimental-vm-modules
```

Comment:
We confirmed with GCP support folks that it's fine and shouldn't be marked experimental in their docs anymore. Do we need this because of the node version we're using locally?

Reply:
@thomik-corp @rhelmer we need this to enable ESM support for Jest. It is only used in testing, so I don't think we need to worry about how it is supported in Firebase. Let me know if that's not the case.

Reply:
I think as long as GCP guys have OK'd it, it should be fine.